### ISSUE TITLE
Focus outline box shadow colour is now true black

### ISSUE THREAD

**Issue by colinrotherham** [alphagov/govuk-frontend#6433]
## Description of the issue

The focus outline `box-shadow` colour is the browser default `#000000`

You need a good monitor to see the difference but I can't tell how long it's been like this

Perhaps it used to use the current text colour at some point in the past?

<img width="1321" height="764" alt="Box shadow colour" src="https://github.com/user-attachments/assets/a5126f3f-eda3-4d15-a78f-8b52d975043d" />

## Steps to reproduce the issue

See the issue when focusing a text input (but not a radio or checkbox)

## Actual vs expected behaviour

I'd expect the `box-shadow` colour to match `$govuk-input-border-colour`, i.e. `#0b0c0c`

Related to https://github.com/nhsuk/nhsuk-frontend/pull/1594 but far more subtle

## Environment (where applicable)

- Operating system: macOS 15.7.2
- Browser: All modern browsers
- Browser version: Safari 26.1, Chrome 142, Firefox 144
- GOV.UK Frontend Version: v5.13.0, v6.0.0-beta.0

**Comment by querkmachine** [comment]
It looks like this is something that has always been the case. Even the absolute earliest version of it in git history has `box-shadow` specified without a colour, thus defaulting to pure black.

https://github.com/alphagov/govuk-frontend/blob/c5af7310105c4c49783a04fed3145008e65b908b/src/govuk/components/input/_input.scss#L27-L41

Given we explicitly set the box shadow colour on the checkboxes and radio buttons, I'm in agreement that this is probably a long-standing oversight. My preliminary checks haven't found that setting it to an explicit colour unduly affects the appearance elsewhere (focused error states override the border colour anyway, and box shadows aren't rendered in forced colour modes in the first place), so this seems like a non-controversial change.
